app/helpers/application_helper.rb

```diff
@@ -33,6 +33,10 @@ def user_signing_up?
     params[:confirmation_token].present? || (current_user && !current_user.two_factor_enabled?)
   end
 
+  def user_registering?
```

Comment:
Should we rename this to `user_on_sign_up_registrations_page?`? Seeing both `user_registering?` and `user_signing_up?` methods is confusion because they both sound like they're checking the same thing.

Reply:
`user_on_sign_up_registrations_page?` seems like a better name for this method.